src/main/scripts/org/reaktivity/specification/nukleus/control/resolve/with.roles/controller.rpt

```diff
@@ -0,0 +1,41 @@
+#
+# Copyright 2016-2017 The Reaktivity Project
+#
+# The Reaktivity Project licenses this file to you under the Apache License,
+# version 2.0 (the "License"); you may not use this file except in compliance
+# with the License. You may obtain a copy of the License at:
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+
+property nukleiR05C ${nuklei:directory("target/nukleus-itests").controlCapacity(1024, 1024)}
+
+property controlR05C ${nukleiR05C.control("security")}
+
+connect "agrona://stream/bidirectional"
+        option agrona:reader ${agrona:broadcastReceiver(controlR05C.controller)}
+        option agrona:writer ${agrona:manyToOneWriter(controlR05C.nukleus)}
+
+connected
+
+write [0x00 0x00 0x00 0x03]
+write ${controlR05C.nextCorrelationId()}
+write [0x05] "role1"
+write [0x05] "role2"
+write flush
+
+read [0x40 0x00 0x00 0x03]
+read ${controlR05C.correlationId()}
+read ([0..2]:securityNukleusId)
+read ([0..6]:roleBits)
```

Comment:
no need to capture values since not used

Reply:
Find that using an external property "authorization" in both scripts since I think the flexibility will be useful (e.g. for testing security nuklei)